config/samples/dataplane_v1beta1_openstackdataplane_baremetal_with_ipam.yaml

```diff
@@ -26,10 +26,20 @@ spec:
           value: "True"
         - name: ANSIBLE_VERBOSITY
           value: "2"
+      preProvisioned: false
+      services:
+        - repo-setup
+        - configure-network
+        - validate-network
+        - install-os
+        - configure-os
+        - run-os
+        - ovn
+        - libvirt
+        - nova
       networkAttachments:
         - ctlplane
       baremetalSetTemplate:
-        deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
```

Comment:
This is required field atm in baremetalset spec, though we use the same secret.

Reply:
oh this was a mistake, as was the addition on line 60

Reply:
Once we move to simplified CRD with NodeSet, we will use webhook to populate this and can remove it then.